src/axolotl/prompt_tokenizers.py

```diff
@@ -67,13 +67,18 @@ def _get_assistant_token(self):
         return False
 
     def _tokenize(self, prompt: str, add_eos_token=True, strip_bos_token=False):
-        result = self.tokenizer(
-            prompt,
-            truncation=True,
-            max_length=self.sequence_len,
-            padding=False,
-            return_tensors=None,
-        )
+        result: Union[Dict[str, List[Union[bool, int]]], BatchEncoding]
+        if not prompt:
```

Comment:
```suggestion
        if not prompt.strip():
```

Not sure if this is necessary.